library/common-library/templates/_license-test-pod.yaml

```diff
@@ -0,0 +1,25 @@
+{{- define "common.tests.license" -}}
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    helm.sh/hook: test-success
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
```

Comment:
didn't we say we were getting rid of this since it was not straightforward to do so?

Reply:
Solved in [#cd009b8](https://github.com/newrelic/helm-charts/pull/713/commits/cd009b8b9c519f4ce67907e67267414b4e5e09de)